### PR TITLE
Cost 1942 ocp on all perspective processing

### PR DESCRIPTION
**Pull request by Red-HAP**
## Jira Ticket

[COST-1942](https://issues.redhat.com/browse/COST-1942)

## Description

Processing changes to switch from materialized views to partitioned tables for the ocp-on-all perspected views 

- reporting_ocpall_compute_summary_pt
- reporting_ocpall_cost_summary_by_account_pt
- reporting_ocpall_cost_summary_by_region_pt
- reporting_ocpall_cost_summary_by_service_pt
- reporting_ocpall_cost_summary_pt
- reporting_ocpall_database_summary_pt
- reporting_ocpall_network_summary_pt
- reporting_ocpall_storage_summary_pt

This pr should also update previously calculated data in the ocpall tables when aws or azure cost models are updated.

## Testing

Data should process properly and numbers should add up properly for OCP on all pages. The pages using the above tables should have data and be correct.

Data processing + a cost model update (AWS and/or Azure) should be reflected in the ocp-on-all views.

The change _should_ be seamless to the UI

### Local

```
================================================================================== short test summary info ===================================================================================
FAILED tests/rest_api/v1/test__cost_model.py::test_api_cost_model_ocp_on_azure_markup_calculation - assert 0.0 != 0.0
FAILED tests/rest_api/v1/test_gcp_cost_reports.py::test_api_gcp_cost_filtered_top_order_by_name[services-asc-two-months-ago-daily] - AssertionError: Api did not return correct top items [...
FAILED tests/rest_api/v1/test_gcp_cost_reports.py::test_api_gcp_cost_filtered_top_order_by_name[services-asc-two-months-ago] - AssertionError: Api did not return correct top items [2021-10]
FAILED tests/rest_api/v1/test_gcp_cost_reports.py::test_api_gcp_cost_filtered_top_order_by_name[services-desc-last-10-days] - AssertionError: Api did not return correct top items [2021-11...
FAILED tests/rest_api/v1/test_gcp_cost_reports.py::test_api_gcp_cost_filtered_top_order_by_name[services-desc-last-30-days] - AssertionError: Api did not return correct top items [2021-10...
FAILED tests/rest_api/v1/test_gcp_cost_reports.py::test_api_gcp_cost_filtered_top_order_by_name[services-desc-last-month-daily] - AssertionError: Api did not return correct top items [202...
FAILED tests/rest_api/v1/test_gcp_cost_reports.py::test_api_gcp_cost_filtered_top_order_by_name[services-desc-last-month] - AssertionError: Api did not return correct top items [2021-11]
FAILED tests/rest_api/v1/test_resource_type_endpoints.py::test_api_gcp_account_search - assert [] != []
FAILED tests/rest_api/v1/test_resource_type_endpoints.py::test_api_gcp_product_search - assert [] != []
FAILED tests/rest_api/v1/test_resource_type_endpoints.py::test_api_gcp_regions_search - assert [] != []
FAILED tests/rest_api/v1/test_resource_type_endpoints.py::test_api_gcp_services_search - assert [] != []
FAILED tests/rest_api/v1/test_resource_type_endpoints.py::test_api_gcp_accounts_compare_data - AssertionError: assert 'example_account_id_3' in []
FAILED tests/rest_api/v1/test_resource_type_endpoints.py::test_api_gcp_projects_compare_data - assert [] != []
FAILED tests/rest_api/v1/test_resource_type_endpoints.py::test_api_gcp_regions_compare_data - assert [] != []
FAILED tests/rest_api/v1/test_resource_type_endpoints.py::test_api_gcp_services_compare_data - assert [] != []
FAILED tests/rest_api/v1/test_source.py::test_api_gcp_source_raw_calc - AssertionError: Raw calculated value does not match report value
============================================== 16 failed, 4449 passed, 696 skipped, 874 deselected, 1 warning, 781 errors in 3478.29s (0:57:58) ==============================================

```

### CODE REVIEW

File: koku/masu/database/sql/reporting_ocpall_compute_summary_pt.sql [hunk 1]
```diff
@@ -0,0 +1,66 @@
+-- OCP ON ALL COMPUTE SUMMARY PROCESSING
```
Comment (maskarb) [inline koku/masu/database/sql/reporting_ocpall_compute_summary_pt.sql:1]: suggestion: With AWS and OCP, we put the partable sql into folders within the `koku/masu/database/sql` directory. Can we move these `ocpall` sql files into `koku/masu/database/sql/openshift/all`?

**Review by maskarb — COMMENTED** [review]
A couple of questions:

File: koku/reporting/models.py [hunk 2]
```diff
@@ -186,6 +194,18 @@
     OCPCostSummaryByNode,
 )
 
+# These are partitioned tables
+OCP_ON_ALL_PERSPECTIVES = (
```
Comment (maskarb) [inline koku/reporting/models.py:198]: nitpick: purely for the sake of consistency with how we've done AWS and OCP tables, let's move this list of models into the `koku/reporting/provider/all/openshift/models.py` file, and import from there.

**Review by maskarb — CHANGES_REQUESTED** [review]
One minor nit-pick, then I think it'll be good to go.

File: koku/reporting/models.py (hunk 2 above)
Comment (Red-HAP) [reply koku/reporting/models.py:198]: @maskarb 
It should be done now

Comment (Red-HAP) [reply koku/reporting/models.py:198]: At least it's consistent as this file has tuples of models and the `models.py` file has tuples of table names

**Review by maskarb — APPROVED** [review]
lgtm!

**Review by maskarb — CHANGES_REQUESTED** [review]
Let's move the migration out to a new PR.

**Review by maskarb — APPROVED** [review]
lgtm! Thanks for moving out those migrations.

**Pull request merged**